### DISCUSSION TITLE
How to connect Fragments easily

### DISCUSSION THREAD

**Discussion by yyh030806** [rdkit/rdkit#8140] [Q&A]
Hi, I want to know how to connect two fragments at the specific site. For example, "[1*]CCC", "[1*]C[2*]" at "[1*]" of the first one and "[2*]" of the second one , and get "CCCC[2*]". Thanks!

**Comment by bp-kelley** [comment]
Use molzip and IsotopeLabels

```
from rdkit import Chem
smi = ".".join(["[1*]CCC", "[1*]C[2*]", "CCCC[2*]"])
m = Chem.MolFromSmiles(smi)

params = Chem.MolzipParams()
params.label = Chem.MolzipLabel.Isotope

res = Chem.molzip(m, params)
print(Chem.MolToSmiles(res))
```

**Reply by bp-kelley** [reply]
Sorry I didn't notice this before.

**Reply by yyh030806** [reply]
Thanks!
